_topic_maps/_topic_map_ms.yml

```diff
@@ -119,23 +119,20 @@ Topics:
   File: understanding-ephemeral-storage-microshift
 - Name: Understanding persistent storage for MicroShift
```

Comment:
nit: If we move "Understanding persistent storage for MicroShift" down so it comes just before "Expanding persistent volumes" then we will have all of the ephemeral storage topics grouped together followed by the persistent storage topics.

Reply:
Yep. I was thinking it, too, but hadn't given the whole thing a read-through. Reshuffled. Going to remove WIP. I think once you're ready, you can lgtm. I'll still want one from @copejon once he's read through ephemeral storage.